lib/node_modules/@stdlib/stats/base/dists/logistic/ctor/benchmark/benchmark.js

```diff
@@ -82,19 +90,24 @@ bench( pkg+'::get:mu', function benchmark( bm ) {
 bench( pkg+'::set:mu', function benchmark( bm ) {
 	var dist;
 	var mu;
+	var len;
 	var s;
 	var y;
 	var i;
 
 	mu = 2.0;
 	s = 3.0;
 	dist = new Logistic( mu, s );
+	len = 100;
+	y = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		y[ i ] = uniform( EPS, 100.0 );
+	}
 
 	bm.tic();
 	for ( i = 0; i < bm.iterations; i++ ) {
-		y = ( 100.0*randu() ) + EPS;
-		dist.mu = y;
-		if ( dist.mu !== y ) {
+		dist.alpha = y[ i % len ];
+		if ( dist.alpha !== y[ i % len ] ) {
```

Comment:
```suggestion
		dist.mu = y[ i % len ];
		if ( dist.mu !== y[ i % len ] ) {
```

The benchmark variables remain the same